Index: str_func.py
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.BaseRevisionTextPatchEP
<+>def to_uppercase(string):\r\n    #  \"\"\"\r\n    # Функция,\r\n    # которая принимает на вход строку и возвращает ее со всеми заглавными буквами \"\"\"\r\n    return string.upper()\r\n
===================================================================
diff --git a/str_func.py b/str_func.py
--- a/str_func.py	
+++ b/str_func.py	
@@ -1,5 +1,19 @@
 def to_uppercase(string):
-    #  """
-    # Функция,
-    # которая принимает на вход строку и возвращает ее со всеми заглавными буквами """
+
+    """
+    Функция, которая принимает на вход строку и возвращает ее со всеми заглавными буквами
+    """
     return string.upper()
+
+
+def capitalize_words(string):
+    """
+    Принимает на вход строку и возвращает новую строку, в которой первые буквы каждого слова заглавные.
+
+    Аргументы:
+    string -- строка, которую нужно изменить
+
+    Возвращаемое значение:
+    Новая строка, в которой первые буквы каждого слова заглавные.
+    """
+    return string.title()
